src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -97,7 +96,7 @@ internal sealed class VariableState
         /// </summary>
         private static readonly TypeSymbolWithAnnotations _invalidType = TypeSymbolWithAnnotations.Create(ErrorTypeSymbol.UnknownResultType);
 
-        private TypeSymbolWithAnnotations _resultType; // PROTOTYPE(NullableReferenceTypes): Should be return value from the visitor, not mutable state.
```

Comment:
Are we just not going to address this?

Reply:
Correct, that's what I suspect at this point.